lib/cisco_node_utils/bgp.rb

```diff
@@ -582,7 +551,7 @@ def default_suppress_fib_pending
     # BGP Timers Getters
     def timer_bgp_keepalive_hold
       match = config_get('bgp', 'timer_bgp_keepalive_hold', @get_args)
-      match.nil? ? default_timer_bgp_keepalive_hold : match.first
+      match.nil? ? default_timer_bgp_keepalive_hold : match
```

Comment:
no, these are always returned together like `[["60", "180"]]` so match.first is really just returning the inner array entry.

This is how they are used:

```
    def timer_bgp_keepalive
      keepalive, _hold = timer_bgp_keepalive_hold
...
    def timer_bgp_holdtime
      _keepalive, hold = timer_bgp_keepalive_hold
```